variants/esp32s3_powerfeather/pins_arduino.h

```diff
@@ -0,0 +1,53 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define USB_VID            0x303A
+#define USB_PID            0x81BB
+#define USB_MANUFACTURER   "PowerFeather"
+#define USB_PRODUCT        "ESP32-S3 PowerFeather"
+#define USB_SERIAL         ""
+
+#define EXTERNAL_NUM_INTERRUPTS 46
+#define NUM_DIGITAL_PINS        23
+#define NUM_ANALOG_INPUTS       13
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<48)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 46)
+
+static const uint8_t ALARM = 7;
+static const uint8_t INT =   5;
+
+static const uint8_t LED =   46;
+static const uint8_t BTN =   0;
+static const uint8_t EN =    13;
+
+static const uint8_t TX =    44;
+static const uint8_t RX =    42;
+static const uint8_t TX0 =   43;
+
+static const uint8_t MISO =  41;
+static const uint8_t MOSI =  40;
+static const uint8_t SCK =   39;
```

Comment:
You are missing SS pin for SPI. If you don't want to specify the pin, you can declare it as -1.